#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_gem_commands.dpatch by  <daigo@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: New patch generated from libgems-ruby 0.8.10-1 diff.gz

@DPATCH@

--- libgems-ruby-0.9.0.orig/lib/rubygems/gem_commands.rb
+++ libgems-ruby-0.9.0/lib/rubygems/gem_commands.rb
@@ -1,4 +1,3 @@
-#!/usr/bin/env ruby
 #--
 # Copyright 2006 by Chad Fowler, Rich Kilmer, Jim Weirich and others.
 # All rights reserved.
@@ -97,6 +96,12 @@
         options[:install_dir] = File.expand_path(value)
       end
 
+      add_option('-B', '--build-root DIR', 
+  'Temporary installation root. Useful for building packages.') do
+  |value, options|
+        options[:build_root] = File.expand_path(value)
+      end
+
       add_option('-d', '--[no-]rdoc', 
 	'Generate RDoc documentation for the gem on install') do
 	|value, options|
